### DISCUSSION TITLE
Just saying hi

### DISCUSSION THREAD

**Discussion by rusefillc** [hyper-tuner/hypertuner-cloud#631] [General]
Why a month without merges? 

Love your log viewers much better than https://rusefi.com/online

**Comment by karniv00l** [comment]
Hi @rusefillc 

Currently finishing up some other projects, hopefully soon I could focus more on this 🤞

**Comment by karniv00l** [comment]
> Love your log viewers much better than https://rusefi.com/online

thanks!

**Comment by pulsatemedia** [comment]
This project looks great, I think it will add a lot of value to the community of folks using tunerstudio! 
Is this something I can deploy now? If yes.. what tech stack do I need? I tried reading some files but it's not super clear to me.

**Reply by karniv00l** [reply]
Hi! 

Not yet, still work in progress 🚧
